web/gds-user-ui/src/components/SwitchFormControl/SwitchFormControl.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { FormControlProps, SwitchProps } from "@chakra-ui/react";
+import { Meta, Story } from "@storybook/react";
+import SwitchFormControl from ".";
+
+interface _FormControlProps extends FormControlProps {
+  formHelperText?: string;
+  controlId: string;
+  label?: string;
+  inputProps?: SwitchProps;
+  name?: string;
+  error?: string;
+  direction?: "column" | "row";
+}
+
+export default {
+  title: "components/ Switch Form Control",
```

Comment:
Would you mind removing the space between words?  It should be "components/SwitchFormControl".